### ISSUE TITLE
[Bug] False throw of `valid-title`

### ISSUE THREAD

**Issue by MartinPilny** [playwright-community/eslint-plugin-playwright#295]
Let's have following test file

```ts
import { TestDetails, test } from '@playwright/test'

test.describe('Suite', () => {
  async function makeTest (testTitle: string, testOptions: TestDetails, urlPath: string) {
    test(testTitle, testOptions, async ({ page }) => {
      await page.goto('www.mywebsite.com/' + urlPath, { waitUntil: 'load' })
    })
  }

  makeTest('Test1', { tag: ['@smoke'] }, 'url_path_1')
  makeTest('Test2', { tag: ['@smoke'] }, 'url_path_2')
  makeTest('Test3', { tag: ['@smoke'] }, 'url_path_3')
})
```

I get this error
![image](https://github.com/playwright-community/eslint-plugin-playwright/assets/45488285/ce852e4b-596b-4555-9a30-033ab34594fd)
which is wrong regarding docs https://github.com/playwright-community/eslint-plugin-playwright/blob/main/docs/rules/valid-title.md

| package | version |
|--------|--------|
| eslint-plugin-playwright | 1.6.2 |
| @playwright/test | 1.44.0 |

**Comment by mskelton** [comment]
@MartinPilny This is not a bug. The Playwright ESLint plugin does not have type information from TypeScript, so it must rely solely on static analysis. Your code is a correct error according to the plugin and should be explicitly ignored with an ESLint disable comment.
